ballerina/client.bal

```diff
@@ -19,17 +19,17 @@ import ballerina/http;
 import ballerina/lang.array;
 import ballerina/time;
 import ballerinax/'client.config;
+import ballerina/url;
 
-# Ballerina Amazon SNS API connector provides the capability to access Amazon Simple Notification Service.
-# This connector lets you to create and manage the sns topics and subscriptions.
+# Ballerina Amazon SNS API connector provides the capability to access Amazon's Simple Notification Service.
+# This connector allows you to create and manage SNS topics and subscriptions.
 #
 # + amazonSNSClient - Connector HTTP endpoint
 # + accessKeyId - Amazon API access key
 # + secretAccessKey - Amazon API secret key
 # + securityToken - Security token
 # + region - Amazon API Region
 # + amazonHost - Amazon host name
-@display {label: "Amazon SNS Client", iconPath: "icon.png"}
```

Comment:
Shall we keep this annotation? any reason to remove

Reply:
Fixed in [020b0a4](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/020b0a40d31509c3e7b499cf2ea5c620bbfc9b2b)